benchmark/bench_parse_call.cpp

```diff
@@ -5,13 +5,90 @@ using namespace benchmark;
 using namespace std;
 
 const padded_string EMPTY_ARRAY("[]", 2);
+const char *TWITTER_JSON = SIMDJSON_BENCHMARK_DATA_DIR "twitter.json";
+const char *GSOC_JSON = SIMDJSON_BENCHMARK_DATA_DIR "gsoc-2018.json";
+
+
+
+static void parse_twitter(State& state) {
+  dom::parser parser;
+  padded_string docdata;
+  simdjson::error_code error;
+  padded_string::load(TWITTER_JSON).tie(docdata, error);
+  if(error) {
+      cerr << "could not parse twitter.json" << error << endl;
+      return;
+  }
+  // we do not want mem. alloc. in the loop.
```

Comment:
I think we do want cold-ish parser buffers in the loop, because we want to test our cache characteristics, and given how most loops will look like `while (get doc) { parser.parse(doc); do a bunch of other stuff; }` it's likely at least some portion (if not all) of the parser internal buffers will be evicted from cache by the time we parse.

I could totally see it as a different test, however. Not needed for this.

Reply:
> I could totally see it as a different test, however. Not needed for this.

Now that we have that, adding 10 different similar tests would be trivial. Copy-paste.